src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random randomCS = new Random();
+    private Color[] colors = Color.values();

```

Comment:
Consider moving the `colors` array initialization directly into the `getRandomColor` method if it's not used anywhere else in the class. This way, you encapsulate the logic within the method that needs it, following the principle of limiting the scope of variables.